lldb/tools/lldb-dap/Protocol/ProtocolTypes.h

```diff
@@ -414,6 +414,16 @@ bool fromJSON(const llvm::json::Value &, SteppingGranularity &,
               llvm::json::Path);
 llvm::json::Value toJSON(const SteppingGranularity &);
 
+/// A Thread.
+struct Thread {
+  /// Unique identifier for the thread.
+  lldb::tid_t id;
```

Comment:
Should this default to `LLDB_INVALID_THREAD_ID`?

Reply:
Done.